yeat/Snakefile

```diff
@@ -28,6 +29,19 @@ rule copyinput:
         copyfile(input.read2, output.read2)
 
 
+rule fastqc:
+    output:
+        out1="seq/fastqc/{sample}_R1_fastqc.html",
+        out2="seq/fastqc/{sample}_R2_fastqc.html"
+    input:
+        read1="seq/input/{sample}_R1.fq.gz",
+        read2="seq/input/{sample}_R2.fq.gz"
+    shell:
+        """
+        fastqc {input.read1} {input.read2} -o seq/fastqc/
+        """
```

Comment:
Only potential comment: might want to add `threads: 2` to the rule and `--threads {threads}` to the shell command for parallel execution.